pytest_robotframework/_internal/pytest/plugin.py

```diff
@@ -314,6 +314,7 @@ def _robot_collect(session: Session):
         "exitonerror": True,
         "prerunmodifier": [RobotSuiteCollector(session)],
         "listener": None,
+        "console": "quiet",
```

Comment:
werent u gonna set it to `"none"` when collecting?

Reply:
errors and warnings sound good

Reply:
ah yes, duplicated errors and warnings don't sound good